app/models/publishing_user.rb

```diff
@@ -0,0 +1,15 @@
+class PublishingUser < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  def self.primary_organisation
+    organisations.order_by('name').first
```

Comment:
why ordered by name? could we instead include a field on the user for the preferred org?